lib/mapper.js

```diff
@@ -19,7 +19,7 @@ var time = require('unix-time');
 exports.track = function(msg, settings){
   var eventName = msg.event();
   var appEvents = settings.appEvents;
-  if (appEvents && appEvents[eventName]) {
+  if (appEvents && appEvents[eventName] && (appEvents[eventName]!=="null") ) {
```

Comment:
ahhh, so this is happening only when they enter a value in the left and then don't choose a mapped value, correct?